### DISCUSSION TITLE
[Community Call] October 25, 2022

### DISCUSSION THREAD

**Discussion by natemoo-re** [withastro/roadmap#356] [Announcements]
## To Discuss

- **Reminder** [Q4 Roadmap](https://github.com/withastro/rfcs/discussions/302)
- **Announcement** Trying out a new `#events` forum in Discord
- **RFC** https://github.com/withastro/rfcs/pull/362
- **Discussion** https://github.com/withastro/rfcs/discussions/355
- **Discussion** https://github.com/withastro/docs/pull/1888

_If there is something that you would like to see discussed, please comment below. We will prioritize items already on the agenda, but if there is time to spare we will open up the call to get attention on and discuss [other threads and feature requests](https://github.com/withastro/rfcs/discussions)._

**Comment by zadeviggers** [comment]
There's been some discussion around adding an opt-in offline mode for the docs site. I've started working on implementing a basic version of this (here's the draft PR where you can view the current state of it: https://github.com/withastro/docs/pull/1888).
I'd like to hear people's thoughts on adding this feature to the docs, any potential issues this could create, and any other comments anyone may have on this (e.g. Should the docs for an MPA framework have a very SPA-related feature? Does this conflict with Astro's core philosophy?).

This isn't related to developing new features in Astro itself, so this could wait until the end of the call.
